### PR TITLE
Updated index template

### PR DESCRIPTION
**Pull request by krumyantsev-sc**

### CODE REVIEW

File: templates/feeds-index.item [hunk 1]
```diff
@@ -1,10 +1,12 @@
       <div class="row mb-4">
```
Comment (achernyakevich-sc) [inline templates/feeds-index.item:1]: Really `mb-4`?

File: templates/feeds-index.item [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
-        <div class="col-md-2 d-flex justify-content-center">
-          <img src="{IMAGE_URL}" alt="{TITLE}" class="img-fluid" />
+        <div class="col-md-2 d-flex justify-content-center align-items-center">
+          <a href="https://smotrim.ru/podcast/{FEED_ID}" target="_blank" style="width: 104px; height: 104px;">
```
Comment (achernyakevich-sc) [inline templates/feeds-index.item]: Do we really need size duplication for `<a>` in the case when nested image already have size?

File: templates/feeds-index.item [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
+            <img src="{IMAGE_URL}" alt="{TITLE}" class="img-fluid" style="width: 104px; height: 104px;" />
```
Comment (achernyakevich-sc) [inline templates/feeds-index.item]: Why we use `style="width: 104px; height: 104px;"` but not attributes `width="104px" height="104px"`?

File: templates/feeds-index.item (hunk 2 above)
Comment (krumyantsev-sc) [reply templates/feeds-index.item]: without size `a` is higher than `img`

File: templates/feeds-index.item (hunk 1 above)
Comment (krumyantsev-sc) [reply templates/feeds-index.item:1]: this is just a margin between rows. Should we remove it?

File: templates/feeds-index.item (hunk 3 above)
Comment (krumyantsev-sc) [reply templates/feeds-index.item]: I will change to attributes

File: templates/feeds-index.item (hunk 1 above)
Comment (krumyantsev-sc) [reply templates/feeds-index.item:1]: image

File: templates/feeds-index.item (hunk 2 above)
Comment (krumyantsev-sc) [reply templates/feeds-index.item]: it is unnecessary here due to `align-items-center`. But without it `a` is higher.

Comment (achernyakevich-sc) [reply templates/feeds-index.item]: Why it is higher? Or wider?

Comment (krumyantsev-sc) [reply templates/feeds-index.item]: Due to `align-items: stretch` set as default value.

Comment (krumyantsev-sc) [reply templates/feeds-index.item]: moving `img` into `a` does not help. HTML ignores any whitespaces between tags. [as i understand:)](https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Whitespace)

**Pull request merged**